source/experimental.html.md.erb

```diff
@@ -2157,7 +2157,7 @@ The type of error that is being reported.
 
 ```
 
-The structured JSON data returned by the data provider, or a representation of the provider's response, to allow PassFort to investigate issues that may arise with your integration.
+The structured JSON data returned by the data provider, or a representation of the provider's response, to allow Passfort to investigate issues that may arise with your integration.
```

Comment:
Remove commas and change "...to investigate issues that may arise..." to "...to investigate any issues that arise..."